python/kserve/kserve/model.py

```diff
@@ -174,16 +174,16 @@ def postprocess(self, response: Union[Dict, ModelInferResponse]) -> Dict:
             return response.get_response(as_json=True)
         return response
 
-    async def _http_predict(self, request: Dict) -> Dict:
+    async def _http_predict(self, request: Dict, headers: Dict[str, str] = None) -> Dict:
         predict_url = PREDICTOR_URL_FORMAT.format(self.predictor_host, self.name)
         if self.protocol == PredictorProtocol.REST_V2.value:
             predict_url = PREDICTOR_V2_URL_FORMAT.format(self.predictor_host, self.name)
-        json_header = {'Content-Type': 'application/json'}
+        # json_header = {'Content-Type': 'application/json'}
```

Comment:
Should this be the default header used when none is provided, to preserve backward compatibility?